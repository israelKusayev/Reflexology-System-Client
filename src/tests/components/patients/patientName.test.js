import React from 'react';
import renderer from 'react-test-renderer';
import PatientName from '../../../components/patient/patientName';
describe('patient name component', () => {
  it('should ', () => {
    const component = renderer.create(
      <patientName
        patient={{ firstName: 'test', lastName: '1234', momName: 'mom' }}
      />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
