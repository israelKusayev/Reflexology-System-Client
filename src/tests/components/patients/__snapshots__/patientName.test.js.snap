// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`patient name component should  1`] = `
<patientName
  patient={
    Object {
      "firstName": "test",
      "lastName": "1234",
      "momName": "mom",
    }
  }
/>
`;
